src/main/java/com/hubspot/jinjava/interpret/TemplateError.java

```diff
@@ -21,8 +21,17 @@
     OTHER
   }
 
+  public enum ErrorItem {
+    TEMPLATE,
+    TOKEN,
+    TAG,
+    FUNCTION,
+    OTHER
```

Comment:
let's add one for `PROPERTY`, and update the missing errors above for `MISSING -> PROPERTY`